packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -367,23 +391,53 @@ const DataFilters = ({
                     {/* Autocomplete */}
                     {filterPopoverCurrentFilter?.variant === "autocomplete" &&
                       filterPopoverCurrentFilter?.options && (
-                        <Autocomplete
-                          label={filterPopoverCurrentFilter.label}
-                          value={
-                            (inputValues[
-                              filterPopoverCurrentFilter.id
-                            ] as string) ?? ""
-                          }
-                          onBlur={function ro() {}}
-                          onChange={function ro() {}}
-                          onFocus={function ro() {}}
-                          onInputChange={function ro() {}}
-                          options={filterPopoverCurrentFilter.options.map(
-                            (option: { label: string }) => ({
-                              label: option.label,
-                            })
-                          )}
-                        />
+                        <Box
+                          sx={{
+                            display: "flex",
+                            gap: 2,
+                            alignItems: "flex-end",
+                          }}
+                        >
+                          <Box sx={{ width: "100%" }}>
+                            <Autocomplete
+                              label={filterPopoverCurrentFilter.label}
+                              value={
+                                inputValues[filterPopoverCurrentFilter.id] ?? ""
+                              }
+                              onChange={(_, value) => {
```

Comment:
It's fine to keep this labeled as `event` even if we don't use it because if we _do_ use it, the `_` isn't very helpful:
```suggestion
                              onChange={(event, value) => {
```

Does ESLint error if we put `event` when it's not used? Might be a rule we'd set at some point.

Reply:
changing to event gives an ESLint error